sphinx-docs/Relationships.md

```diff
@@ -0,0 +1,50 @@
+# Relationships
+
+Many CALDERA abilities require input variables called "facts" to be provided before the ability can be run. These facts can either be provided manually through the CALDERA server GUI, or they can be discovered by a previous ability. 
+
+## Creating Relationships using Abilities
+
+### Example
+
+As an example, the following printer discovery ability will create two variables called `host.print.file` and `host.print.size`:
```

Comment:
let's use "facts" instead of "variables" to stay consistent with our terminology